docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -176,6 +176,10 @@ Avoid feature lists with more than 10 items, they are difficult to scan and make
 
 Apps should have at least one screenshot in english. For medium complexity apps 3-6 is a good range, for very large apps with lots of screens 6-10 is appropriate. More than 10 is probably too many.
 
+:::tip
+Your best screenshot should be on the first position.
```

Comment:
I'd maybe add some kind of reasoning, e.g. "... because this is the screenshot that will be used in featured banners."

Also, I've encountered some apps whose screenshots are technically fine but the first one is a mobile size. Perhaps we should have something about making sure the first screenshot is a desktop size?

Reply:
I don't think we can make sure, there are apps that mostly target mobile I think?

Reply:
Maybe, but that's a very small minority of apps. In practice the ones I've tried look so bad that I don't want to feature them on banners, which is not a good state of affairs. Maybe we could have a way for reviewers to pick which screenshot to feature medium-term, or would that be overkill?

Reply:
I think that might end up confusing? Clicking an app with a banner and then seeing a different screenshot of the app with a mobile layout?

So I would much prefer to have the best screenshot first, which I guess I would agree should be non mobile then.

Can you prepare some words?